braindecode/preprocessing/preprocess_classes.py

```diff
@@ -0,0 +1,155 @@
+"""Preprocessor objects based on mne methods.
+"""
+
+# Author: Bruna Lopes <brunajaflopes@gmail.com>
+#
+# License: BSD-3
+
+import mne.io
+from braindecode.util import _update_moabb_docstring
+from braindecode.preprocessing import Preprocessor
+
+
+class Resample(Preprocessor):
+    doc = """
+    Subclass of Preprocessor to perform resampling using mne.Epochs/mne.io.Raw's resample method.
+
+    Parameters
+    ----------
+    kwargs:
+        Keyword arguments to be passed to mne.Epochs's resample method.
```

Comment:
Could be nice to include a link to the resample webpage:
```suggestion
        Keyword arguments to be passed to `mne.Epochs's resample <https://mne.tools/stable/generated/mne.filter.resample.html>`__ method.
```